app/services/onlinetesting/Phase2TestService.scala

```diff
@@ -356,6 +372,9 @@ trait Phase2TestService extends OnlineTestService with Phase2TestConcern with Sc
     }
   }
 
+  protected[onlinetesting] def calculateAbsoluteTimeWithAdjustments(application: OnlineTestApplication): Int = {
+    (application.eTrayAdjustments.flatMap { etrayAdjustments => etrayAdjustments.timeNeeded }.getOrElse(0) * 80 / 100) + 80
```

Comment:
We pass the time extension as a percentage, so if we pass 25%, for a base duration of 80 minutes, we mean 25 * 80 / 100 = 20 extra minutes.